src/store/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware, compose } from "redux";
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/index';
+
+const storeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(
+    rootReducer, 
+    storeEnhancers(applyMiddleware(thunk)),
+    );
+
+
+export default store
```

Comment:
@ElMonstro, kindly add an EOF here.

Reply:
@abulojoshua1  I have done the requested change